torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -394,16 +398,18 @@ def affine_bounding_box(
     center: Optional[List[float]] = None,
 ) -> torch.Tensor:
     original_shape = bounding_box.shape
-    bounding_box = convert_format_bounding_box(
-        bounding_box, old_format=format, new_format=features.BoundingBoxFormat.XYXY
+    bounding_box = (
+        bounding_box.clone()
```

Comment:
Are you sure we need clone here? I don't see any inplace ops in `_affine_bounding_box_xyxy`. Does it make sense to remove it?

Reply:
I was going for parity, i.e. replicating the old behavior exactly. But you are right indeed that we don't need to clone here due to the absence of inplace ops. Since this is a perf improvement, let me add this for this and all other comments you added.